bld/CLMBuildNamelist.pm

```diff
@@ -366,7 +366,7 @@ sub check_for_perl_utils {
     } else {
```

Comment:
This won't work.  The cime directory will always exist, instead you should look for a file in the cime directory, for example -d "$cimeroot/CIME"

Reply:
This seems to me outside the scope of updating the documentation. 
@ekluzek pls let me know what you think about this one.

Reply:
this has not been answered, so I will open an issue